### PR TITLE
feat(docs): Inital theme for docs

### PR DESCRIPTION
**Pull request by sebald**
- removed hardcoded dependency to b2b theme
- removed all usage of `styles`

### CODE REVIEW

**Comment by sebald** [comment]
@ti10le @viktoria-schwarz not perfect, but I couldn't stand the hardwired usage of the b2b theme 😝

File: docs/src/theme/index.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    background: colors.gray[80],
+    primary: colors.brand.red,
+    secondary: colors.brand.orange,
+    disabled: colors.gray[70],
+    error: colors.red,
+    warning: colors.yellow,
+    info: colors.blue,
+    success: colors.green,
+  },
+  radii: {
```
Comment (ti10le) [inline docs/src/theme/index.ts:29]: do you really mean radii?

File: docs/src/wrapper.tsx [hunk 2]
```diff
@@ -16,7 +16,17 @@ export const WrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
 export const WrapRootElement: GatsbyBrowser['wrapRootElement'] = ({
   element,
 }) => (
-  <ThemeProvider theme={b2bTheme}>
```
Comment (ti10le) [inline docs/src/wrapper.tsx:19]: After this removal our components doesn't have the b2b-theme style like before. Do you did this in regard of my switch-theme functionality?

File: docs/src/wrapper.tsx [hunk 3]
```diff
@@ -16,7 +16,17 @@ export const WrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
...
+  <ThemeProvider theme={theme}>
     <MDXProvider components={mdxComponents}>{element}</MDXProvider>;
   </ThemeProvider>
 );
+
+/**
+ * Enforce reloading to update styles.
```
Comment (ti10le) [inline docs/src/wrapper.tsx:25]: Can you tell me why we need this?

**Review by ti10le — APPROVED** [review]

**Review by ti10le — CHANGES_REQUESTED** [review]
change from approve to request changes

File: docs/src/theme/index.ts (hunk 1 above)
Comment (sebald) [reply docs/src/theme/index.ts:29]: Yes, see https://theme-ui.com/theme-spec

Comment (viktoria-schwarz) [reply docs/src/theme/index.ts:29]: plural of radius right?

File: docs/src/wrapper.tsx (hunk 2 above)
Comment (sebald) [reply docs/src/wrapper.tsx:19]: No, I just switched to the docs theme :)

Comment (sebald) [reply docs/src/wrapper.tsx:19]: The theme from our docs app shouldn't leak into the examples in the first place.

File: docs/src/wrapper.tsx (hunk 3 above)
Comment (sebald) [reply docs/src/wrapper.tsx:25]: Webpack HMR does not recognize changes in the styles (e.g. when you update a theme and this causes changes in the CSS styles).

This makes sure the page automatically refreshes, so we don't have to do it manually.

File: docs/src/theme/index.ts (hunk 1 above)
Comment (sebald) [reply docs/src/theme/index.ts:29]: I guess so? I just read the spec 😂

File: docs/src/wrapper.tsx (hunk 3 above)
Comment (ti10le) [reply docs/src/wrapper.tsx:25]: Thanks :)

**Review by ti10le — DISMISSED** [review]

**Comment by viktoria-schwarz** [comment]
@sebald changeset is missing :)

**Comment by sebald** [comment]
@viktoria-schwarz done! :P

**Review by ti10le — APPROVED** [review]

**Pull request merged**